templates/web/pages/product_edit/product_edit_form_display.tt.html

```diff
@@ -96,12 +96,16 @@
 
         <!--nutrient fieldset-->
         <div class="fieldset" id="nutrition"><legend>[% lang('nutrition_data') %]</legend>
+            <!-- empty hidden checkbox with the same name so that the browser sends a value when the box is unchecked -->
+            <input type="hidden" name="no_nutrition_data" value="" />
```

Comment:
Hum this does not work for me !

If I check the box, this is not taken into account.

In Food.pm, in `assign_nutriments_values_from_request_parameters`, `param("no_nutrition_data")` is `''`